### PR TITLE
added authentication

### PR DESCRIPTION
**Pull request by ashishnnnnn**
-> Added Login Page.
-> Added Signup Page.
-> Added Validation of signup page input.
-> Added a test credential login O

----------------------------------------------------------------------------------------------------------------------------------------------------------------------

Screenshot from 2022-04-09 15-31-37

----------------------------------------------------------------------------------------------------------------------------------------------------------------------

Screenshot from 2022-04-09 15-31-45
ption.

----------------------------------------------------------------------------------------------------------------------------------------------------------------------

Screenshot from 2022-04-09 15-31-57

### CODE REVIEW

File: src/pages/Signup/Signup.jsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+export const Signup = () => {
+  let navigate = useNavigate();
+  const { auth_state, setAuthState } = useAuthContext();
+  const { user } = auth_state;
+  const { firstName, lastName, email, password, confirmPassword } = user;
+  const { handleaddtoast } = useToast();
+  const signupHandle = async () => {
+    console.log("HELLO");
+    try {
```
Comment (Manpreet-hub) [inline src/pages/Signup/Signup.jsx:15]: can remove console.

**Review by Manpreet-hub — APPROVED** [review]
Code looks good and well structured

File: src/Components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -6,18 +6,88 @@ import { useState } from "react";
...
+                  className="btn btn-primary cursor-pointer"
+                >
+                  LogOut
+                </div>
+              </div>
+            ) : (
+              <div className="side-hidder-info pad-t-1 pad-l-1">
+                <div
+                  onClick={(e) => {
+                    e.stopPropagation();
```
Comment (tanishk15sharma) [inline src/Components/Navbar/Navbar.jsx]: Why you are using `e.stopPropagation()` ?

**Review by tanishk15sharma — APPROVED** [review]

File: src/Components/Navbar/Navbar.jsx (hunk 2 above)
Comment (ashishnnnnn) [reply src/Components/Navbar/Navbar.jsx]: That was not needed..I was using stopPropagation for some of the click handler...So Accidentally i was here too..

File: src/pages/Signup/Signup.jsx (hunk 1 above)
Comment (ashishnnnnn) [reply src/pages/Signup/Signup.jsx:15]: Removed

**Pull request merged**